ext/rinku/utf8.c

```diff
@@ -106,6 +130,24 @@ int32_t utf8proc_rewind(const uint8_t *data, size_t pos)
 	return read_cp(&data[pos - length], length);
 }
 
+int32_t utf8proc_open_paren_character(int32_t cclose)
+{
+	switch (cclose) {
+	case '"': return '"';
+	case '\'':  return '\'';
+	case ')': return '(';
+	case ']': return '[';
+	case '}': return '{';
+	case 65289: return 65288; /* （） */
+	case 12305: return 12304; /* 【】 */
+	case 12303: return 12302; /* 『』 */
+	case 12301: return 12300; /* 「」 */
+	case 12299: return 12298; /* 《》 */
+	case 12297: return 12296; /* 〈〉 */
```

Comment:
I've just added those ones which is normally used in Japanese, so tell me if you need more to add